WPFHospitalEditor/Repository/IMapObjectRepository.cs

```diff
@@ -11,7 +11,8 @@ interface IMapObjectRepository
         MapObject update(MapObject mapObject);
         void saveAll(List<MapObject> entities);
         List<MapObject> getAll();
-        List<MapObject> getOutterMapObjects(List<MapObject> allMapObjects);
+        List<MapObject> getOutterMapObjects();
         MapObject findMapObjectById(int id);
+        void setAllSelectedFieldsToFalse();
```

Comment:
This method should not be in repository interface. The repository should contains only the things that the repository is expected to fulfill. Any additional data processing should be performed at a service.

Reply:
This method doesn't exist anymore, because I deleted selected filed from MapObject and it doesn't need to be set to false after exiting application.